### ISSUE TITLE
Any Duna module doesnt pull power from Duna PDU or pull local supplies

### ISSUE THREAD

**Issue by RookFett** [UmbraSpaceIndustries/MKS#1437]
Setup (clean install MKS Constellation - latest version of KSP 1.5.1)
Duna PDU configured, generating power, has Ranger cooling, engineer and pilot on board.

Problem
Duna Pioneer parked within 10m of PDU, no power transfer, PAW states that there are no PDU's in range (power coupler).  Will run out eventually while running.

Interesting thing:
Another Duna Pioneer parked withing 10 m of PDU, with a Microwave Power Transceiver placed on the module, sees the PDU, and is pulling power.  _(this as expected)_ However, the power coupler on the module PAW states no PDU in range.

**Comment by RookFett** [comment]
tested all the duna modules - they all seem not to pull power unless there is a microwave pt on them.  Side note  - the modules wont scavenge supplies if I don't add the microwave, not sure if this is connected?

Had 3 PDU's running, all the other modules, once the microwave was installed, pull supplies and had happy kerbals, however, the PDU's didn't have a microwave, so they had unhappy kerbals.  Once I added microwaves to the PDU's - they filled up with supplies and everyone was happy.

**Comment by RookFett** [comment]
Here is a save - stock with USI/MS constellation - [https://1drv.ms/u/s!AngGcYPkeU3shLEu6iBZ79LTdZvnNw](url)

Made 4 habitats off the runway - 3 with microwave pt's on them, the fourth one does not have one.  dropped kontainer (wh enabled) with supplies,  the 3 pulled in, the fourth did not.    Using hyperedit, I drained the kontainer, and one of the 3 habs with supplies, and it immediately did a local pull from the other two habs.     Since I had removed KIS and KAS from this save, I could not move a microwave pt to the one missing it.  (however, in other games with KIS/KAS - as soon as I place it - the module local scavenges).

Power also will not work till I put the microwave pt on modules.   Is this a new design or bug?

**Comment by RookFett** [comment]
Not sure if I am doing something wrong or not understanding what to do - but I tried to make the refinery example shown on this wiki for scavenging - using a miner with a refinery - and it would not pull the minerals mined to make chemicals.  I've included the save game with the modules I made.  When I fill the tanks with Hyperedit - everything works, but when I try to make the refinery pull the minerals - it does not.  strangely - when I drop a supplies kontainer - it will pull them - but will not pull minerals.   https://1drv.ms/u/s!AngGcYPkeU3shLE3HD5Q0vR7T79dlw
is the link to the save game.

[Mining Test Alpha.zip](https://github.com/UmbraSpaceIndustries/MKS/files/2683256/Mining.Test.Alpha.zip)
[default.zip](https://github.com/UmbraSpaceIndustries/MKS/files/2683257/default.zip)
